packages/kbn-i18n/src/core/i18n/intl_polyfill.js

```diff
@@ -17,12 +17,6 @@
  * under the License.
```

Comment:
__issue:__ looks like intl polyfill is out of scope for this particular PR, can we add it in a separate PR and not mix two unrelated changes?

Reply:
Yes, actually you are absolutely right, but I added because of 2 reasons:
1. It is really small change and easy to review, but if I would add it in another PR, it would be blocked.
2. In this PR I added support for different envs. And would like to deliver fully working code without `Intl` issues

Reply:
> It is really small change and easy to review

Hmm, can't agree here, it's a new dependency and we should know and discuss all the consequences it may have (e.g. build size). And it's for node-only, does it block anything specific right now in `Index Management` localization effort?

> but if I would add it in another PR, it would be blocked.

That's totally fine, let's solve issues when they arrive.

Reply:
@azasypkin you are right. moved this logic